core/src/main/java/org/mskcc/cbio/portal/model/converter/MutationModelConverter.java

```diff
@@ -45,8 +45,8 @@ public ExtendedMutation convert(Mutation mutation) {
         if (mutationEvent.getEndPosition() != null) {
             event.setEndPosition(mutationEvent.getEndPosition());
         }
-        if (mutationEvent.getAminoAcidChange() != null) {
-            event.setProteinChange(mutationEvent.getAminoAcidChange());
+        if (mutationEvent.getProteinChange() != null) {
```

Comment:
all these if-nulls feel very anti-patterny, maybe mutation events should be modelled differently? Could it have several subclasses, or maybe more of an EAV-like lookup like the one for clinical attributes?

I understand this may not be the focus of this PR though, it's just to put it out there!